model/src/main/java/io/github/droidkaigi/feeder/Locale.kt

```diff
@@ -0,0 +1,9 @@
+package io.github.droidkaigi.feeder
+
+actual fun getDefaultLocale(): Locale =
+    if (java.util.Locale.getDefault() == java.util.Locale.JAPAN) {
+        Locale.JAPAN
+    } else {
+        Locale.OTHER
+    }
+
```

Comment:
Could you remove this empty line? 🙏 

![image](https://user-images.githubusercontent.com/5106629/109782704-84a9a080-7c4c-11eb-940f-5f061468fbf5.png)

Reply:
@tomoya0x00 
I fix it!